example/tasks.toml

```
@@ -26,3 +26,28 @@
     done
   """
 
+# these examples illustrate the behavior of env vars in tasks
+#
+# the output of the two following tasks should be the same except for the TESTENV line
+# the first task should be empty, the second should have the TESTENV line with the
+# value "test". Both should include the env vars from the parent process.
+[[task]]
+  id = "noenv"
+  desc = "with no env var provided, the task inherits (sic) the env from the parent process"
+  type = "short"
+  cmd = """
+    env | head -n 25
```

```suggestion
    echo ENVIRONMENT_TEST: $ENVIRONMENT_TEST
    echo "there are $(env | wc -l) environment variables"
```

I don't like printing the content of env because,
- it doesn't identify to readers which aspects are salient
- it may leak credentials into test logs